nf_core/pipeline-template/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
```

Comment:
```suggestion
# Config file for Python. Mostly used to configure linting of bin/check_samplesheet.py with Black.
# Should be kept the same as nf-core/tools to avoid fighting with template synchronisation.
[tool.black]
```